src/test/java/co/rsk/bitcoinj/script/RedeemScriptParserFactoryTest.java

```diff
@@ -237,4 +238,36 @@ public void create_RedeemScriptParser_object_from_hardcoded_testnet_redeem_scrip
 
         Assert.assertEquals(MultiSigType.NO_MULTISIG_TYPE, parser.getMultiSigType());
     }
+
+    @Test
+    public void createRedeemScriptParser_whenPassingStandardScriptSig_shouldReturnStandardRedeemScriptParser() {
```

Comment:
```suggestion
    public void createRedeemScriptParser_forStandardMultiSigInputScript_shouldReturnStandardRedeemScriptParser() {
```

Reply:
btw, for this test it applies the same as [here](https://github.com/rsksmart/bitcoinj-thin/pull/93/files#diff-337ff6a4185b238d1e0cca58fdf6b2b02227928a54c3242bb4fa616c543d2828R254-R255) right?

Reply:
Done